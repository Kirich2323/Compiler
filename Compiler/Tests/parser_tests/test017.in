1 shr 2