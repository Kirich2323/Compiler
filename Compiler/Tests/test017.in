'simple string'
''
'string with'#10' espace symbols'''