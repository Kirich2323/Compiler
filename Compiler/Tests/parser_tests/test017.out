\- shr
   |- 1
   \- 2